cmd/lifecycle/analyzer.go

```diff
@@ -20,51 +21,71 @@ import (
 
 type analyzeCmd struct {
 	//flags: inputs
-	cacheDir      string
-	cacheImageTag string
-	groupPath     string
-	uid, gid      int
 	analyzeArgs
+	uid, gid int
+
+	cacheDir      string // Platform API < 0.7
+	cacheImageTag string // Platform API < 0.7
+	groupPath     string // Platform API < 0.7
 
 	//flags: paths to write data
 	analyzedPath string
 }
 
 type analyzeArgs struct {
 	//inputs needed when run by creator
-	imageName  string
-	layersDir  string
-	skipLayers bool
-	useDaemon  bool
-
-	platform cmd.Platform
-
-	//construct if necessary before dropping privileges
-	docker   client.CommonAPIClient
-	keychain authn.Keychain
+	imageName   string
```

Comment:
I feel like the comment above `//inputs needed when run by creator` might have lost its original context. Isn't `imageName` for example always needed?

Reply:
I agree the comment is maybe not 100% correct or as clear anymore.

Reply:
Let's take it out then. We could add something back if / when we become confused.